### PR TITLE
Feature/add guest

### PR DESCRIPTION
**Pull request by schnika**
allows other users to add guests with the `/gast Max Mustermann` command.

todo

- [x] tests
- [x] linting

### CODE REVIEW

File: src/addGuest.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  });
+
+  if (alreadyInList) {
+    const output = userName + "ist schon beim Training " + spreadsheetName +
+      " dabei. Du brauchst sie/ihn also nicht mehr einzutragen! :white_check_mark: :woman-running: :runner: ";
+    return generateResultMessage(output, false);
+  }
+  addParticipantToSpreadsheet(spreadsheet, userName, userId);
+
+  let output = userName + "ist als Gast am " + spreadsheetName + " dabei :confetti_ball:";
```
Comment (lennartblom) [inline src/addGuest.js:71]: Missing space after `userName`.

File: src/addGuest.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  return generateResultMessage(output, true);
```
Comment (lennartblom) [inline src/addGuest.js:72]: Also:
It turns out this `guest` behaviour is exposed when normal participants are joining the upcoming training. 

See latest comment in Slack
Screenshot 2020-10-28 at 10 43 19
.

**Review by lennartblom — CHANGES_REQUESTED** [review]

File: src/addGuest.js (hunk 2 above)
Comment (schnika) [reply src/addGuest.js]: Sadly the addGuest implementation of addNewParticipantToSpreadsheet "shadowed" the implementation defined in "participate.js". I extracted spreadsheet specific methods to a spreadsheet service.

**Comment by schnika** [comment]
@lennartblom tests mache ich heute Abend :+1:

**Pull request merged**